.gitignore

```diff
@@ -188,18 +188,15 @@ UpgradeLog*.htm
 # Microsoft Fakes
 FakesAssemblies/
 
+#NETStandard
+*.lock.json
+
 # Data generated from unit tests
-NetTopologySuite.Samples.Shapefiles/ZMtest.shp
-NetTopologySuite.Samples.Shapefiles/ZMtest.shx
-NetTopologySuite.Samples.Shapefiles/ZMtest.dbf
-NetTopologySuite.Samples.Shapefiles/__empty.dbf
-NetTopologySuite.Samples.Shapefiles/__empty.shp
-NetTopologySuite.Samples.Shapefiles/__empty.shx
+/NetTopologySuite.Samples.Shapefiles/
 /NetTopologySuite/Algorithm/HCoordinate_DD.cs
 /NetTopologySuite/Algorithm/CGAlgorithmsAP.cs
 /NetTopologySuite.IO/NetTopologySuite.IO.GeoTools/NetTopologySuite.IO.GeoTools.csproj.DotSettings
-NetTopologySuite.Samples.Shapefiles/tmp*.shp
-NetTopologySuite.Samples.Shapefiles/tmp*.shx
-NetTopologySuite.Samples.Shapefiles/tmp*.dbf
-NetTopologySuite.Samples.Shapefiles/test_arcview.shp
-NetTopologySuite.Samples.Shapefiles/test_buffer.shp
```

Comment:
Surely, at the very least, **this line** was doing something for us?  As evidenced by the change to this file further below...